<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mypolyn</class>
 <widget class="QDialog" name="Mypolyn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="SubEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>265</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>一元多项式计算</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="MultiEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ClearEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入变量名 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ResultEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>525</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="DeriButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>395</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>多项式求导</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SubEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>265</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEdit3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>存储结果变量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>清空多项式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>265</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>多项式减法</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DestroyEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入变量名 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InitEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入变量名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MultiEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SubEdit3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>265</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>存储结果变量</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LookEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>135</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入变量名 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="DestroyButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>460</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>销毁多项式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InitEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>请输入变量值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>525</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>结果</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>多项式加法</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AddEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InitButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>定义多项式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LookButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>135</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看多项式值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DeriEdit1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>395</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DeriEdit2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>395</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>参与运算变量 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="MultiButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>多项式乘法</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择实现方式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>17</y>
     <width>160</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>顺序表</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>顺序表</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>链表</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>输入说明：以&quot;系数,次数&quot;的形式，系数可以为浮点
数，次数为自然数，次数0也需标出,每对数之间
用空格隔开，例：1,0 2,3表示1+2x^3。</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="MultiEdit3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>存储结果变量</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
